<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogColorBalanceFilter</class>
 <widget class="QWidget" name="DialogColorBalanceFilter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>470</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Color balance</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_preset">
     <item>
      <widget class="QLabel" name="label_presetTitle">
       <property name="text">
        <string>Strength</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_presetButtons">
       <item>
        <widget class="QRadioButton" name="radioButton_balanceLight">
         <property name="toolTip">
          <string>Fastest option with a subtle smoothing effect.</string>
         </property>
         <property name="text">
          <string>Light</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton_balanceMedium">
         <property name="toolTip">
          <string>Balanced mode.</string>
         </property>
         <property name="text">
          <string>Medium</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton_balanceStrong">
         <property name="toolTip">
          <string>Stronger smoothing with longer calculation time.</string>
         </property>
         <property name="text">
          <string>Strong</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_parameters">
     <item row="0" column="0">
      <widget class="QLabel" name="label_kMin">
       <property name="text">
        <string>k min</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSpinBox" name="spinBox_kMin">
       <property name="minimum">
        <number>4</number>
       </property>
       <property name="maximum">
        <number>200</number>
       </property>
       <property name="value">
        <number>24</number>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_kMax">
       <property name="text">
        <string>k max</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="spinBox_kMax">
       <property name="minimum">
        <number>4</number>
       </property>
       <property name="maximum">
        <number>200</number>
       </property>
       <property name="value">
        <number>40</number>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_trim">
       <property name="text">
        <string>Trim (%)</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QDoubleSpinBox" name="doubleSpinBox_trim">
       <property name="minimum">
        <double>0.000000000000000</double>
       </property>
       <property name="maximum">
        <double>40.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>1.000000000000000</double>
       </property>
       <property name="value">
        <double>18.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_mode">
     <property name="title">
      <string>Balance mode</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_mode">
      <item>
       <widget class="QRadioButton" name="radioButton_separate">
        <property name="toolTip">
         <string>Apply the balance to each scan separately.</string>
        </property>
        <property name="text">
         <string>Separate balance</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_global">
        <property name="toolTip">
         <string>Apply the balance to each scan while taking overlaps into account.</string>
        </property>
        <property name="text">
         <string>Global balance</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBox_balanceIntensityRGB">
     <property name="text">
      <string>Apply on RGB and i</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_folder">
     <item row="0" column="0">
      <widget class="QLabel" name="label_folder">
       <property name="text">
        <string>Folder</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEdit_folder">
       <property name="placeholderText">
        <string>Enter output folder</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QToolButton" name="toolButton_folder">
       <property name="text">
        <string>Select Out Folder</string>
       </property>
       <property name="icon">
        <iconset resource="../../../openscantools.qrc">
         <normaloff>:/icons/100x100/open_project.png</normaloff>:/icons/100x100/open_project.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBox_openFolderAfterExport">
     <property name="text">
      <string>Open folder after export</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_buttons">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_ok">
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_cancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../../../openscantools.qrc"/>
 </resources>
 <connections/>
</ui>
